src/interfaces/graph-update-job.interface.ts

```diff
@@ -0,0 +1,20 @@
+import { MessageSourceId, ProviderId } from '@frequency-chain/api-augment/interfaces';
+
+// Note: DSNP IDs are u64 on Frequency, but since JS 'bigint' doesn't automatically
+//       serialize to JSON, we use strings here.
```

Comment:
This made me realize I need to pass strings all the way to neon layer and parse, Would call a PR on that

Reply:
Good to know this!